paddle/phi/api/yaml/generator/api_base.py

```diff
@@ -1317,22 +1323,11 @@ def gene_api_code(self):
                     api_code = ""
                 api_code = api_code + self.gene_base_api_code(inplace_flag=True)
             return api_code
-
+        elif self.is_invoke_composite_api:
+            # for composite and invoke api, dygraph use prim::xxx_grad method
+            return ''
         else:
-            invoke_func_name = self.invoke.split('(')[0].strip()
-            if invoke_func_name in self.attrs['names']:
-                # Adjust the param whose name is same with api invoked.
-                pattern = r'\W' + invoke_func_name + '[^A-Za-z0-9_(]'
-
-                def adjust_name(matched):
-                    matched_str = matched.group()
-                    return matched_str[0:-1] + '_val' + matched_str[-1]
-
-                invoke_code = re.sub(pattern, adjust_name, self.invoke)
-                params_code = re.sub(
-                    pattern, adjust_name, self.get_define_args()
-                )
-            else:
-                invoke_code = self.invoke
-                params_code = self.get_define_args()
```

Comment:
only embedding_grad generate paddle_api , other invoke api don't need generate paddle_api。
this branch only for scale whose attrs is same as api name, but scale_grad don't generate paddle_api